### PR TITLE
Accept URL parameters

### PR DESCRIPTION
**Pull request by ascott1**
This glues together the URL parameter work that @mistergone had done with the rest of the JS.
## Screenshot

screen shot 2015-12-17 at 3 54 29 pm
## Bonus (aka PR scope creep)
- Fixes the linting task and addresses all of the linting errors
- Calculates the remaining cost in UI and model
## Review

@niqjohnson & @marteki

### CODE REVIEW

**Comment by ascott1** [comment]
It's worth noting that those parameters may need to change. I know @higs4281 has documented what they should be and those changes are easy to make.

File: src/disclosures/js/models/financial-model.js [hunk 1]
```diff
@@ -7,8 +7,9 @@ var financialModel = {
   values: {},
 
   init: function() {
-    var val = getViewValues.inputs();
-    this.calc ( val );
+    var val = getViewValues.init();
+    console.log( val );
```
Comment (niqjohnson) [inline src/disclosures/js/models/financial-model.js]: Get rid of this `console.log`, or do we need to keep it in there for now?

**Comment by niqjohnson** [comment]
This is looking good. One small line comment above and a couple of questions. I used these parameters:

```
?tf=20000&rb=1000&bk=500&tr=500&oe=500&sc=2000&pg=2000&sv=800&ws=800&fm=800&fl=3000&pk=3000&ss=1500&su=1500&il=500&ir=4&pl=1000&pr=6
```

Almost all the parameters populated the expected fields except for a couple of oddball ones:
- Should `pr` populate the private loan interest rate field?
- `il` and `ir` map to "loans from your school" and the associated interest rate in the current version of the tool. Should those map to "tuition payment plan from your school" and the associated interest rate in this version?

If we want to address those in a future PR, I say :+1: to this one.

**Comment by higs4281** [comment]
:+1: to linting. i vote to rename vars after we get this on the boards.

File: src/disclosures/js/models/financial-model.js (hunk 1 above)
Comment (ascott1) [inline src/disclosures/js/models/financial-model.js]: Oh, the shame.

**Comment by ascott1** [comment]
@niqjohnson the changes I pushed should address your comments and we can rename variables as needed going forward.

**Comment by niqjohnson** [comment]
:+1:

All the parameters in the URL are populating all the right fields for me. Awesome!

**Comment by ascott1** [comment]
:tada:

**Pull request merged**